Super_Team_Picker/routes/cohorts.js

```diff
@@ -0,0 +1,192 @@
+const express = require('express');
+const knex = require('../db/client');
+const router = express.Router();
+
+function shuffle(array) {
+    for (var i = array.length - 1; i > 0; i--) {
+        var j = Math.floor(Math.random() * (i + 1));
+        var temp = array[i];
+        array[i] = array[j];
+        array[j] = temp;
+    }
+}
+
+router.get('/', (req, res) => {
+    knex("cohorts")
+    .select("*")
+    .orderBy("id", "asc")
+    .then((data) => {
+        res.render("cohorts/index", {
+            pageTitle: "Team List",
+            cohorts: data,
+        });
+    });
+});
+
+router.get('/new', (req, res) => {
+    res.render("cohorts/edit", {
+        pageTitle: "New Team Picker",
+        cohort: null
+    });
+});
+
+router.post('/', (req, res) => {
+    const cohortParmas = {
+        members: req.body.members,
+        name: req.body.name,
+        logoUrl: req.body.logoUrl
+    };
+    knex("cohorts").insert(cohortParmas).returning("id").then(data => {
+        res.redirect(`/cohorts/${data[0]}`);
+    })
+});
+ 
+
+router.get('/:id', (req, res) => {
+    if(isNaN(parseInt(req.params.id))){
+        res.render('cohorts/show',{cohort:false,pageTitle: "Not Found - Super Team Picker"});
+    }else{     
+        knex("cohorts")
+            .select("*")
+            .where({
+                id: req.params.id
+            })
+            .then(data => {
+                if(data===undefined){
+                    res.render('cohorts/show',{cohort:false,pageTitle: "Not Found - Super Team Picker"})
+                }else{
+                    let memberList = null;
+                    let isTeamCount = 'checked';
+                    let isMemberCount = null;                
+                    let members = data[0].members.split(',');
+                    let inputNumb = null;                         
+                    if (req.query.method && req.query.quantity) {
```

Comment:
to safe-guard that this operation will not fail, you can do `req.query && (req.query.method && req.query.quantity)`

Reply:
thanks